igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/TravisBuildMonitor.groovy

```diff
@@ -254,4 +256,11 @@ class TravisBuildMonitor implements PollingMonitor{
     private int buildCacheJobTTLSeconds() {
         return TimeUnit.DAYS.toSeconds(cachedJobTTLDays)
     }
+
+    private List<Repo> filterOutOldBuilds(List<Repo> repos){
+        Long threshold = new Date().getTime() - TimeUnit.DAYS.toMillis(cachedJobTTLDays) + BUILD_STARTED_AT_THRESHOLD
```

Comment:
This could benefit from a comment explaining why the threshold needs the additional offset beyond the `cachedJobTTLDays`.